hack/verify-lint.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# standard bash error handling
+set -o nounset # treat unset variables as an error and exit immediately.
+set -o errexit # exit immediately when a command fails.
+set -E         # needs to be set if we want the ERR trap
+
+readonly CURRENT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+readonly ROOT_PATH="${1:-$( cd "${CURRENT_DIR}/.." && pwd )}" # first argument or root of the project
+readonly TMP_DIR=$(mktemp -d)
+readonly GOLANGCI_LINT_VERSION="v1.38.0"
+
+source "${CURRENT_DIR}/utilities.sh" || { echo 'Cannot load CI utilities.'; exit 1; }
+
+cleanup() {
+    rm -rf "${TMP_DIR}" || true
+}
+
+trap cleanup EXIT SIGINT
+
+golangci::install() {
+  export PATH="${INSTALL_DIR}:${PATH}"
+
+  shout "Install the golangci-lint in version ${GOLANGCI_LINT_VERSION}"
+  curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b "${INSTALL_DIR}" ${GOLANGCI_LINT_VERSION}
+  echo -e "${GREEN}√ install golangci-lint${NC}"
+
+}
+
+golangci::run_checks() {
+  shout "Run golangci-lint checks"
+  LINTS=(
+    # default golangci-lint lints
+    deadcode errcheck gosimple govet ineffassign staticcheck \
+    structcheck typecheck unused varcheck \
+    # additional lints
+    golint gofmt misspell gochecknoinits unparam scopelint gosec
+  )
+
+  ENABLE=$(sed 's/ /,/g' <<< "${LINTS[@]}")
```

Comment:
Why the variable is named `ENABLE`.
I would named it as `LINTS`.

Reply:
Because it contains the value we pass to the `--enable` flag.
`LINTS` would also be ok of course.

Reply:
I think `LINTS` name is better, because `ENABLE` suggests: `I am a bool`, but in reality I am not :(

Reply:
Thing is, 2 lines above we already have the variable named `LINTS` with all the linters in a readable format.
`ENABLE` is just the `LINTS` variable with a format that the `--enable` flag will accept.

I will try to just get rid of the `ENABLE` variable and do the parsing inline.

Reply:
Got rid of the `ENABLE` var, so issue solved ;)

Reply:
Hi @dbadura, if you do not have other comments, could you please approve the PR?
Thank you.